### PR TITLE
build: use `make testSuite` to ensure all testSuites enabled

### PR DESCRIPTION
**Pull request by SunRunAway**
### What problem does this PR solve? 

use `make testSuite` to ensure all testSuites enabled

### What is changed and how it works?

add a bash script in ./tools/check/check_testSuite.sh

### Check List 

Tests 

 - Unit test
 
Code changes

 - None

Side effects

 - The script runs a little slowly. 

Related changes

 - None

### CODE REVIEW

File: store/tikv/backoff_test.go [hunk 1]
```diff
@@ -37,6 +37,6 @@ func (s *testBackoffSuite) TearDownTest(c *C) {
 func (s *testBackoffSuite) TestBackoffWithMax(c *C) {
 	b := NewBackoffer(context.TODO(), 2000)
 	err := b.BackoffWithMaxSleep(boTxnLockFast, 30, errors.New("test"))
-	c.Assert(err, NotNil)
+	c.Assert(err, IsNil)
```
Comment (zz-jason) [inline store/tikv/backoff_test.go]: why this test result is changed?

Comment (SunRunAway) [reply store/tikv/backoff_test.go]: This test case fails but never runs once before :(

**Comment by zz-jason** [comment]
How about cherry-pick this commit to the release branch?

**Comment by zz-jason** [comment]
@SunRunAway Please follow [CONTRIBUTING.md](https://github.com/pingcap/tidb/blob/master/CONTRIBUTING.md) to refine the PR title.

**Pull request merged**